Update Knowledge Base post re MIDI Mixer control (#1119)
jamulussoftware/jamulussoftware.github.io: _posts/2022-02-01-Midi-Mixerboard-Control.md

<!--more-->

## What do you need?

> 1. Jamulus with JACK support (not the standard ASIO version if you are using Windows, _but see the note below regarding Jamulus versions 3.12.0 onwards_), you can [download the Jamulus JACK version for Windows here]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})).

Reviewer: ```suggestion
1. Jamulus (prior to version 3.12.0 you will need the [JACK version]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }}), for which you will also need JACK for Windows from [JACK Audio](https://jackaudio.org/downloads/)).
```
and drop point 4 below.

Reviewer: In fact, this page becomes a bit redundant.

It says "Here's how to set up Jamulus for JACK".  We should have that covered elsewhere.

It says "Here's how to use Jamulus with `--ctrlmidich` and we definitely have that covered elsewhere.

The only remaining thing it says that's not said elsewhere is that - _prior to 3.12.0_ - you'd need the JACK version on Windows for `--ctrlmidich`... and I think it would be better to put that on [the `--ctrlmidich` page](https://jamulus.io/wiki/Tips-Tricks-More#using-ctrlmidich-for-midi-controllers).

[That](https://jamulus.io/wiki/Tips-Tricks-More#using-ctrlmidich-for-midi-controllers) needs updating to say "Jamulus since 3.7.0 (except Windows ASIO), updated with further functionality in ... and supported on Windows ASIO since 3.12.0" -- probably in a new paragraph.

Maybe this is easier to reference and read: but maybe then we need to separate `--ctrlmidich` into its own page that gets translated, like [Tips-Tricks-More](https://jamulus.io/wiki/Tips-Tricks-More) does.

The more I think about it, the more that appeals.

2. You need an audio device with ASIO support or you can use a universal ASIO driver like ASIO4ALL.
3. You need a MIDI Control Surface. Many different models are available. I am using a Worlde EasyControl 9.
 ![worlde-easycontrol-9](https://user-images.githubusercontent.com/13550012/152056661-a446f24e-0598-4164-bc0f-a2510ab33c5e.png)

> 4. JACK for Windows [get it from the JACK audio page](https://jackaudio.org/downloads/).

Reviewer: See above.

### Update for Jamulus 3.12.0 on Windows

> The ASIO version of Jamulus 3.12.0 for Windows has introduced native support for MIDI devices without the need for JACK. If using 3.12.0 or later on Windows, you can skip all the sections below that relate to JACK. [See here](/wiki/Tips-Tricks-More#using---ctrlmidich-for-midi-controllers) for more details.

Reviewer 2: ```suggestion
The ASIO version of Jamulus 3.12.0 for Windows has introduced native support for MIDI devices without the need for JACK. If using 3.12.0 or newer on Windows, you can skip all the sections below that relate to JACK. [This topic has more details](/wiki/Tips-Tricks-More#using---ctrlmidich-for-midi-controllers).
```

Reviewer: Pedantically...

> 3.12.0 or later
> 3.12.0 or newer

should probably be

> version 3.12.0 or more recent
